testslide/__init__.py

```diff
@@ -175,6 +214,15 @@ def sub_example(self, name=None):
         with self._sub_examples_agg_ex.catch():
             yield
 
+    def async_run(self, coro):
```

Comment:
If we think about TestSlide test runner being able to also run `unittest.TestCase` tests, it is far to argue that having asyncio health checks there would be cool as well, so maybe we should also have it at `testslide.TestCase`.

Also, I can picture users asking "why use self.asyncio_run and not asyncio.run?", so I'd consider calling this method something like `asyncio_run_with_health_checks`  (or whatever).